modules/porous_flow/test/tests/sinks/PorousFlowPiecewiseLinearSink_BC_eg1.i

```diff
@@ -0,0 +1,131 @@
+## This is an example input file showing how to set a Type I BC with PorousFlowPiecewiseLinearSink
```

Comment:
Oh, i just realised that you probably don't want to include it in the test/tests/sinks/tests file, otherwise you will have to modify that file with a lot of SQA stuff.  Please still include a gold file (suggestions below) and i'll do the tests file later.